tests/cpp_unit_tests/test_three_phase_tensor.cpp

```diff
@@ -85,7 +85,7 @@ TEST_CASE("Three phase tensor") {
         CHECK((mat4 == mat).all());
         CHECK(diag_mult(1.0, 2.0, 3.0) == 6.0);
         // test layout
-        double* arr = (double*)&mat1;
+        auto* arr = (double*)&mat1;
```

Comment:
it is possible that it wants you to use `reinterpret_cast` or `static_ptr_cast` here, not sure. if it starts complaining, just ignore it for now